src/MUIDataTable.js

```diff
@@ -241,11 +241,12 @@ class MUIDataTable extends React.Component {
     },
     showResponsive: false,
     searchText: null,
+    isPrinting: false,
   };
 
   constructor() {
     super();
-    this.tableRef = false;
+    this.tableRef = React.createRef();
```

Comment:
Why is this being changed?

Reply:
This because the update of react and material ui versions

Reply:
ah, I see the 16.3 migration docs on this, you are correct. Thank you for spotting this!